### DISCUSSION TITLE
Some Issues on 3D Flow Field Inversion

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#672] [General]
I am conducting flow field inversion for square pipeline flow, I have made changes based on the periodic mountain example, but the following errors have occurred：
[logOpt.txt](https://github.com/user-attachments/files/16749472/logOpt.txt)
[1] --> FOAM FATAL ERROR: 
[1] betaFieldInversion not found in volScalar and volVector Fields 
How can I solve this problem

**Comment by friedenhe** [comment]
Your runScript set a wrong fieldName `betaFieldInversion` Please refer to the ramp tutorial to set a correct fieldName, which can be `betaFINuTilda` or `betaFIOmega`
